examples/docker-compose-counter-v3.yaml

```diff
@@ -0,0 +1,20 @@
+version: "3"
+
+services:
+
+ web:
+  image: tuna/docker-counter23
+  ports:
+    - "5000:5000"
+  deploy:
+    restart_policy:
+      condition: any
+  labels:
+    kompose.service.type: NodePort
+
+ redis:
+  image: redis:3.0
+  deploy:
+     replicas: 2
```

Comment:
this *still* wont work.. it won't sync or anything on Kubernetes. You have to remove `deploy: replicas: 2` altogether. Do you know what I mean? 

Adding replicas: 2 won't magically make it work on Kubernetes with creating a Redis cluster, you'll have to manually create redis-master and redis-slave.

For example, this would be valid / work:
```
  redis-master:
    image: gcr.io/google_containers/redis:e2e 
    ports:
      - "6379"

  redis-slave:
    image: gcr.io/google_samples/gb-redisslave:v1
    deploy:
     replicas: 2
    ports:
      - "6379"
    environment:
      - GET_HOSTS_FROM=dns
```